server/chat/backend/agent/skills/integrations/slack/SKILL.md

```
@@ -0,0 +1,47 @@
+  - get_channel_history
+  - get_thread_replies
+index: "Slack messaging -- list channels, read messages, read threads"
+rca_priority: 50
+metadata:
+  author: aurora
+  version: "1.0"
+---
+
+# Slack Tools
+
+## Overview
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)